src/hub/interfaces/IHubRequestManagerCallback.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+import {PoolId} from "../../common/types/PoolId.sol";
+import {AssetId} from "../../common/types/AssetId.sol";
+import {ShareClassId} from "../../common/types/ShareClassId.sol";
+
+interface IHubRequestManagerCallback {
+    /// @notice Handles a request callback originating from a spoke request.
```

Comment:
NIT. Not sure if this doc line is accurate, this comes from hubRequestManager and goes to spoke